rpcs3/rpcs3qt/trophy_notification_frame.cpp

```diff
@@ -43,14 +43,14 @@ trophy_notification_frame::trophy_notification_frame(const std::vector<uchar>& i
 	QString trophyType = "";
 	switch (trophy.trophyGrade)
 	{
-	case SCE_NP_TROPHY_GRADE_BRONZE:   trophyType = "bronze";   break;
-	case SCE_NP_TROPHY_GRADE_SILVER:   trophyType = "silver";   break;
-	case SCE_NP_TROPHY_GRADE_GOLD:     trophyType = "gold";     break;
-	case SCE_NP_TROPHY_GRADE_PLATINUM: trophyType = "platinum"; break;
+	case SCE_NP_TROPHY_GRADE_BRONZE:   trophyType = tr("Bronze");   break;
+	case SCE_NP_TROPHY_GRADE_SILVER:   trophyType = tr("Silver");   break;
+	case SCE_NP_TROPHY_GRADE_GOLD:     trophyType = tr("Gold");     break;
+	case SCE_NP_TROPHY_GRADE_PLATINUM: trophyType = tr("Platinum"); break;
 	default: break;
 	}
 
-	trophyName->setText(tr("You have earned the %1 trophy.\n").arg(trophyType) + qstr(trophy.name));
+	trophyName->setText(tr("You have earned the %1 trophy.\n%2").arg(trophyType).arg(trophy.name));
```

Comment:
Maybe it would be better to create 4 translations for this, since translating "Gold" itself might not make much sense without the entire sentence in other languages

Reply:
I'm open to this possibility.  Although it potentially makes a little more work for translators if there are the individual terms used around the place (but I don't recall that being anywhere).
I might need to defer to a more lingual expert than myself.  It is definitely perhaps a bit on the nose pushing just the adjective to the translators, although I do suspect that within our rpcs3 world the terms 'Bronze', 'Silver', 'Gold', 'Platinum' are pretty unique identifiers.  I'm not even sure if they 'should' get translated.  I've never played non-English PS3.

Reply:
I don't think it's more work to translate 4 almost identical strings than to translate 4 strings + 1 string with an additional formatting. 😉

Reply:
I agree.  But if we use 'Gold' in like four or five places.. and then suddenly each one of those places becomes a custom string.. that's a change from 4x precious metals + 4-5x sub-in places => 16-20x unique strings to translate... which might be a little more annoying for them.
But again, happy to hear from the translators as to which is easiest/bestest for them

Reply:
I don't see it being used in 4 or five places here 🧐

Reply:
@IlDucci as one of the 'translation people(s)', do you have an opinion?

Do we want a full translatable string for each trophy type
i.e. "You have earned the Platinum trophy.%1" (trophy.name)
... "You have earned the Gold trophy.%1" ... Silver ... Bronze

or is it acceptable to have the following translation strings
"You have earned the %1 trophy.%2" (trophy colour, trophy.name)
"Platinum", "Gold", "Silver", "Bronze"

Reply:
In some languages, you have to use different cases for these words in different contexts. For example in Finnish if you're just listing the trophy somewhere you use the nominative case *kultainen*, but when earning you use accusative case *kultaisen*. In english you can just slap words together without much consideration, but when same principle is used in languages like Finnish it looks really sloppy.

Reply:
I think is much more appropiate to have full translatable strings for that case rather than having it piecemeal. Using the appropiate tools (like QTLinguist), the Translation Memory will probably remind translators how the first entry was translated.

Reply:
That'll teach me for asking :P